pkgs/development/python-modules/latex2pydata/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, setuptools
+, nix-update-script
+}:
+
+buildPythonPackage rec {
+  pname = "latex2pydata";
+  version = "0.2.0";
+
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-lFYGBFox7fv/vlfqZN3xsh9UIRCQ+C5Cizq9j4RTcJ0=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+  ];
+
+  passthru.updateScript = nix-update-script { };
+
```

Comment:
Missing `pytestCheckHook` (put it at `nativeCheckInputs`).

Reply:
Done.